quiz.js

```diff
@@ -0,0 +1,50 @@
+var button = document.getElementById("submitButton");
+button.addEventListener("click", handler);
+
+var numInput = document.getElementById("num");
+numInput.addEventListener("keyup", handler);
+
+var charInput = document.getElementById("char");
+charInput.addEventListener("keyup", handler);
+
+
+
+//Checks if both values are present - Places values into Object
+function handler(event) {
+	event.preventDefault();
+	// console.log("test", event);
```

Comment:
For final submission/merging, commented out code should be removed, also on lines 23-25